boards/openmote-b/include/board.h

```diff
@@ -77,6 +77,15 @@
 #define RF24_SWITCH_AT86RF215_TOGGLE (RF_SWITCH_PORT->DATA ^=  RF24_SWITCH_AT86RF215_MASK)
 /** @} */
 
+/**
+ * @name    RF CORE observable signals settings
+ * @{
+ */
+#define CONFIG_CC2538_RF_OBS_SIG_0_PCX  4   /* PC4 */
```

Comment:
```suggestion
#define CONFIG_CC2538_RF_OBS_SIG_0_PCX  5   /* PC5 */
```

PC4 is already LED0 which is the default LED used by applications

Reply:
Will do.